README.md

```diff
@@ -8,6 +8,13 @@ This repository contains the Dockerfiles for Objectif Libre's official images.
 
 Contains the Dockerfile for CloudKitty's API and its dependencies.
 
+For the configuration file, you can use :
+
+- `/etc/cloudkitty/cloudkitty.conf`
+- `/etc/cloudkitty-config/cloudkitty.conf`
+
+*If you want to deploy this application on kubernetes, the second location could be useful.*
```

Comment:
please explain why

Reply:
+ I'd rather have only one way to do things. eg; "Mount your custom cloudkitty.conf and metrics.yml files to `/xxx`" .

Users usually do weird things when they have a choice :wink:

Reply:
standardize this could be a good idea